docs/content/en/docs/installation/runtime-hooks.md

```diff
@@ -132,44 +90,108 @@ minikube cp /tmp/new-config.toml /etc/containerd/config.toml
 minikube ssh sudo systemctl restart containerd
 ```
 
+{{% /tab %}}
+
+{{< /tabpane >}}
+
 ### Install Tetragon
 
-```shell
+{{< tabpane lang=shell >}}
+{{< tab "CRI-O (oci-hooks)" >}}
+helm install \
+   --namespace kube-system \
+   --set rthooks.enabled=true \
+   --set rthooks.interface=oci-hooks \
+   tetragon ./install/kubernetes/tetragon
+{{< /tab >}}
+{{< tab "Containerd (nri-hook)" >}}
 helm install \
    --namespace kube-system \
-   --set tetragon.image.override=docker.io/cilium/tetragon:latest \
-   --set tetragonOperator.image.override=docker.io/cilium/tetragon-operator:latest \
    --set rthooks.enabled=true \
    --set rthooks.interface=nri-hook \
-   --set rthooks.image.override=docker.io/cilium/tetragon-rthooks:latest \
    tetragon ./install/kubernetes/tetragon
-```
+{{< /tab >}}
+{{< /tabpane >}}
 
-```shell
-kubectl -n kube-system get pods | grep tetragon
+```shel
+kubecl -n kube-system get pods | grep tetragon
 ```
 
-Output should be similar to:
+With output similar to:
 ```
-tetragon-operator-754b85cfd4-2mdd7   1/1     Running   0              24m
-tetragon-pjrsf                       2/2     Running   0              24m
-tetragon-rthooks-6g8cq               1/1     Running   0              24m
+tetragon-hpjwq                       2/2     Running   0          2m42s
+tetragon-operator-664ddc8957-9lmd2   1/1     Running   0          2m42s
+tetragon-rthooks-m24xr               1/1     Running   0          2m42s
 ```
 
-### Test
+### Test Runtime Hooks
 
 Start a pod:
-
 ```shell
 kubectl run test --image=debian  --rm -it -- /bin/bash
 ```
 
-Examine the log file:
+Check logs:
 ```shell
 minikube ssh 'tail -1 /opt/tetragon/tetragon-oci-hook.log'
 ```
 
 Output:
 ```json
-{"time":"2024-07-02T12:02:02.823291054Z","level":"INFO","msg":"hook request to agent succeeded","hook":"createRuntime","start-time":"2024-07-02T12:02:02.816185835Z","req-cgroups":"/kubepods/besteffort/pod9305570c-ac68-4f95-96d8-afbb138bd0b0/42469ae2c52d0ee340b550b8a07a142c9b8cc709aa8ca75b777bb00812149621","req-rootdir":"/run/containerd/io.containerd.runtime.v2.task/k8s.io/42469ae2c52d0ee340b550b8a07a142c9b8cc709aa8ca75b777bb00812149621","req-containerName":"test"}
+{"time":"2024-07-01T10:57:21.435689144Z","level":"INFO","msg":"hook request to agent succeeded","hook":"create-container","start-time":"2024-07-01T10:57:21.433755984Z","req-cgroups":"/kubepods/besteffort/podd4e74de2-0db8-4143-ae55-695b2489c727/crio-828977b42e3149b502b31708778d0c057efbce038af80d0882ed3e0cb0ff8796","req-rootdir":"/run/containers/storage/overlay-containers/828977b42e3149b502b31708778d0c057efbce038af80d0882ed3e0cb0ff8796/userdata","req-containerName":"test"}
+```
+
+## Configuring Runtime Hooks installation
+
+### Installation directory (`installDir`)
+
+For tetragon runtime hooks to work, a binary (`tetragon-oci-hook`) needs to be installed on the
+host. This happens by the runtime hooks. This binary is installed in `/opt/tetragon` by default.
```

Comment:
"This happens by the runtime hooks"

I think a word may be missing there

Reply:
There was not :) rewarded to make it clearer. Thanks!